lib/Entity/Video.php

```diff
@@ -411,4 +417,16 @@ public function setUploadFileName(string $uploadFileName): self
 
         return $this;
     }
+
+    public function setCorporateTubeMetadata(CorporateTubeMetaData $corporateTubeMetadata): self
```

Comment:
but where this setter is called? in another ticket?

Reply:
I don't know :-) It's a library, maybe somebody needs it.